modules/contact/test/tests/3d-mortar-contact/tests

```diff
@@ -83,7 +83,7 @@
                   'using nodal-attached geometry and a frictional pressure vector generated by an '
                   'auxiliary kernel through a user-friendly action. Results are diffed against '
                   'non-action output'
-    rel_err = 1.0e-5
+    rel_err = 3.0e-5
```

Comment:
Do we still need this after your expanded ghosting  work?

Reply:
Most likely. We do not ghost point neighbors unless one of our mortar nodal aux kernels is present, so in theory the ghosting in this particular simulation should be totally unchanged.